assets/_scss/core/_variables.scss

```diff
@@ -2,7 +2,7 @@
 $em-base:             10px;
 $base-font-size:      rem(17px);
 $small-font-size:     rem(14px);
-$lead-font-size:      rem(22px);
+$lead-font-size:      rem(20px);
```

Comment:
So I checked the visuals and the default one is 22px bc the default uses sans serif lead text. So I think this should be changed back and the 20px one added in styleguide.